src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -140,4 +141,30 @@ public function exportThemeAction(Request $request)
 
         return $response->setContentDisposition(ResponseHeaderBag::DISPOSITION_ATTACHMENT);
     }
+
+    /**
+     * Show translations settings page
+     *
+     * @Template("@PrestaShop/Admin/Improve/International/Translations/translations_settings.html.twig")
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller')~'_')", message="Access denied.")
+     *
+     * @param Request $request
+     *
+     * @return array
+     */
+    public function showSettingsAction(Request $request)
+    {
+        $legacyController = $request->attributes->get('_legacy_controller');
+        $kpiRowFactory = $this->get('prestashop.core.kpi_row.factory.translations_page');
+        $presenter = $this->get('prestashop.core.kpi_row.presenter');
+
+        $presentedKpiRow = $presenter->present($kpiRowFactory->build());
+
+        return [
+            'layoutTitle' => $this->trans('Translations', 'Admin.Navigation.Menu'),
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink($legacyController),
+            'kpiRow' => $presentedKpiRow,
```

Comment:
as KPI stuff is present in multiples pages, can we create a specific action to render the KPI block in `CommonController` instead?